templates/partial/filter.html

```diff
@@ -10,7 +10,9 @@ <h2 class="govuk-heading-m">Filter</h2>
       </div>
     </div>
     <div class="moj-filter__content">
-      {% include "partial/selected_filters.html" %}
+      {% if label_clear_href|get_keys|length > 0 %}
```

Comment:
[minor] Should we rename `label_clear_href` to something more descriptive?

Reply:
I have renamed to `remove_filter_hrefs` for clarity.